sdk/python/pulumi_aws/amplify/domain_association.py

```diff
@@ -69,6 +73,18 @@ def sub_domains(self) -> pulumi.Input[Sequence[pulumi.Input['DomainAssociationSu
     def sub_domains(self, value: pulumi.Input[Sequence[pulumi.Input['DomainAssociationSubDomainArgs']]]):
         pulumi.set(self, "sub_domains", value)
 
+    @property
+    @pulumi.getter(name="enableAutoSubDomain")
+    def enable_auto_sub_domain(self) -> Optional[pulumi.Input[bool]]:
+        """
+        Enables the automated creation of subdomains for branches.
+        """
+        return pulumi.get(self, "enable_auto_sub_domain")
```

Comment:
Very awkward that a getter / property has a side-effecty name.

Reply:
As a field the name makes sense, because it's something that holds a value; naming the method for the field is awkward. Maybe codegen should have a special case for `enable` and `disable`.